packages/vue-components/src/Searchbar.vue

```diff
@@ -276,7 +274,13 @@ export default {
         left: auto;
     }
 
-    .hidden {
+    .placeholderDiv-hidden {
```

Comment:
```suggestion
    .placeholder-div-hidden {
```

I don't have much context here, just dropping a not-so-helpful minor nit 🙈

Reply:
Thanks! Updated :) Also added `overflow: hidden` to prevent the placeholder div from overflowing the navbar.